.evergreen/build-manylinux.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -ex
+
+docker version
+
+# 2020-03-20-2fda31c Was the last release to include Python 3.4.
+images=(quay.io/pypa/manylinux1_x86_64:2020-03-20-2fda31c \
```

Comment:
Do all all versions other than 3.4 get built twice?

Reply:
Yes. They get built first with these old images and then rebuilt and overwritten with the latest manylinux1_x86_64/manylinux1_i686 images.